### PR TITLE
Rudimentary tests for not covered entry points

### PR DESCRIPTION
**Pull request by yu-tian113**
Add tests for ReactART Circle, Rectangle, Wedge components.

### CODE REVIEW

**Comment by yu-tian113** [comment]
Hi @gaearon , this is the follow up PR for #11750 , changes are
- Add tests to ReactART `Circle`, `Rectangle`, and `Wedge`
- Add redirections for source testing

I've also searched the codebase for entry points not covered in tests. 
- All entry points are covered in **source testing**
- `react-reconciler/reflection` and `react-dom/unstable-native-dependencies` are not covered in **build testing**

A question regarding the `Circle`, `Rectangle`, and `Wedge` components, are they considered a part of ReactART? Why we put them in `npm` but `src`? And should we refactor them with ES6 syntax to align with ReactART?

Thanks.

File: packages/react-art/src/__tests__/ReactART-test.js [hunk 1]
```diff
@@ -329,3 +334,103 @@ describe('ReactART', () => {
...
+  });
+
+  it('should turn degrees to radians', () => {
+    const wedgeInstance = renderer
+      .create(
+        <Wedge outerRadius={50} startAngle={0} endAngle={360} fill="blue" />,
+      )
+      .getInstance();
+    expect(wedgeInstance._degreesToRadians(0)).toBe(0);
+    expect(wedgeInstance._degreesToRadians(180)).toBe(3.141592653589793);
```
Comment (gaearon) [inline packages/react-art/src/__tests__/ReactART-test.js]: I don’t think we should be calling internal functions. Just using props is enough.

Comment (yu-tian113) [reply packages/react-art/src/__tests__/ReactART-test.js]: ok, these tests are removed.
Can't find a good way to test the math generating `path`, all other props are covered in snapshot tests.
Please let me know if anything's missing.

**Comment by gaearon** [comment]
Thanks!

**Pull request merged**